sklearn/datasets/svmlight_format.py

```diff
@@ -129,16 +142,20 @@ def _gen_open(f):
         return open(f, "rb")
 
 
-def _open_and_load(f, dtype, multilabel, zero_based, query_id):
+def _open_and_load(f, dtype, multilabel, zero_based, query_id,
+                   offset=0, length=-1):
     if hasattr(f, "read"):
-        return _load_svmlight_file(f, dtype, multilabel, zero_based, query_id)
+        return _load_svmlight_file(f, dtype, multilabel, zero_based,
+                                   query_id, offset, length)
     # XXX remove closing when Python 2.7+/3.1+ required
     with closing(_gen_open(f)) as f:
-        return _load_svmlight_file(f, dtype, multilabel, zero_based, query_id)
+        return _load_svmlight_file(f, dtype, multilabel, zero_based,
+                                   query_id, offset, length)
 
 
 def load_svmlight_files(files, n_features=None, dtype=np.float64,
-                        multilabel=False, zero_based="auto", query_id=False):
+                        multilabel=False, zero_based="auto", query_id=False,
+                        offset=0, length=-1):
```

Comment:
Just curious, does it actually make sense to have the same offset and length when calling this with multiple files?